oml/lightning/entrypoints/validate.py

```diff
@@ -11,9 +11,14 @@
 from oml.lightning.modules.retrieval import RetrievalModule
 from oml.metrics.embeddings import EmbeddingMetrics
 from oml.registry.models import get_extractor_by_cfg
+from oml.utils.misc import dictconfig_to_dict
 
 
 def main(cfg: TCfg) -> Tuple[pl.Trainer, Dict[str, Any]]:
+    print(cfg)
```

Comment:
hydra config should be resolved before print, so use `print` after `dictconfig_to_dict(cfg)`